src/assets/scripts/embed.njk

```diff
@@ -3,7 +3,14 @@ permalink: 'embed.js'
 ---
 const WEBRING_DATA_URL = "https://xyquadrat.ch/polyring/data/members.json"
```

Comment:
If we migrate to `{{meta.dataUrl}}` (which sounds like a good idea), we can drop this, since it's not used anymore.

Reply:
Ah had that removed in another edit. I'll push that change, too